components/appbar/BasicAppBar.component.jsx

```diff
@@ -104,6 +104,12 @@ class BasicAppBar extends React.Component {
               disabled={mappingButton[ViewName.HOME] === false}
               icon={<Home />}
             />
+            <BottomNavigationAction
+              label="Donation"
+              value={ViewName.DONATION}
+              disabled={mappingButton[ViewName.DONATION] === false}
+              icon={<Paid />}
+            />
```

Comment:
We can put this button to the left?

Reply:
You want `donation` button first then home button?

Reply:
yes, because in material style the home button is on the middle fo the UI